src/main/webapp/i18n/de/textExercise.json

```diff
@@ -29,7 +29,17 @@
             "checkPlagiarism": "Auf Plagiate prüfen",
             "exampleSubmissionsRequireExercise": "Um eine Beispielabgabe zu erstellen, musst du zunächst die Erstellung der Textaufgabe abschließen. Du wirst automatisch weitergeleitet.",
             "saveSuccessful": "Dein Antwort wurde erfolgreich gespeichert!",
-            "error": "Ein Fehler ist aufgetreten. Bitte versuche es später noch einmal."
+            "error": "Ein Fehler ist aufgetreten. Bitte versuche es später noch einmal.",
+            "tutorEffortStatistics": {
+                "title": "Tutor Aufwand Statistik",
+                "numberOfSubmissionsAssessed": "Anzahl der insgesamt bewerteten Einreichungen beteiligt",
+                "totalTimeSpent": "Gesamtdauer der Benotung (in Minuten)",
+                "averageTime": "Benotungsrhythmus (Submissions/Min)",
```

Comment:
Rather use Minuten or Min
It's not really "Rhythmus", it's rather "Bewertungsgeschwindigkeit"